contracts/test/Move.lock

```diff
@@ -21,7 +21,7 @@ dependencies = [
 ]
 
 [move.toolchain-version]
-compiler-version = "1.55.0"
+compiler-version = "1.50.1"
```

Comment:
can we revert this?
i think you just need to update your SUI CLI version locally. sth like `brew upgrade sui`